opr/oprencoding/opr.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package oprencoding;
+
+message ProtoOPRMin {
```

Comment:
I don't think the Min suffix is necessary anymore

Reply:
I can drop the Min.